ssl/ssl_sess.c

```diff
@@ -916,6 +916,13 @@ long SSL_SESSION_get_time(const SSL_SESSION *s)
     return (long)ossl_time_to_time_t(s->time);
 }
 
+long SSL_SESSION_get_rtt(const SSL_SESSION *s) // return rtt in microseconds. Matches the units of nginx's $tcpinfo_rtt
```

Comment:
Please do not use C++ style comments. Also the description should be above the function.